src/Main.java

```diff
@@ -1,5 +1,62 @@
+import org.w3c.dom.ls.LSOutput;
+
+import java.time.LocalDate;
+
 public class Main {
+    public static boolean determineYear(int year) {
```

Comment:
```suggestion
    public static boolean isLeapYear(int year) {
```
методы, которые возвращают boolean, лучше называть на is